libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -1400,32 +1408,26 @@ private Object getBuildConfigValue(Context context, String fieldName) {
     }
 
     /**
-     * Returns if dark theme should be displayed.  This takes the Android system state and the options
-     * to disable for force dark theme into account.
-     * @param activity     Activity used to check the system dark theme state.
+     * Returns if dark theme should be displayed.  The value is retrieved from the OS, is no value is set.
+     * @see SalesforceSDKManager#setTheme
+     *
      * @return             True if dark theme should be displayed, otherwise false.
      */
-    public boolean isDarkTheme(Activity activity) {
-        int currentNightMode = activity.getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
-        boolean systemDarkTheme = currentNightMode == Configuration.UI_MODE_NIGHT_YES;
-
-        return !darkThemeDisabled && (systemDarkTheme || forceDarkTheme);
-    }
-
-    /**
-     * Disables dark theme for the SDK.
-     * @param disabled     True if dark theme should be disabled.
-     */
-    public synchronized void disableDarkTheme(boolean disabled) {
-        darkThemeDisabled = disabled;
+    public boolean isDarkTheme() {
+        if (theme == Theme.SYSTEM_DEFAULT) {
+            int currentNightMode = context.getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
+            return currentNightMode == Configuration.UI_MODE_NIGHT_YES;
+        } else {
+            return theme == Theme.DARK;
+        }
     }
 
     /**
-     * Forces dark theme for all supported API levels
-     * @param forced     True if dark theme should be force enabled.
+     * Sets the theme for the SDK.  This value only persists as long as the instance of SalesforceSDKManager.
+     * @param theme     The theme to use.
      */
-    public synchronized void forceDarkTheme(boolean forced) {
-        forceDarkTheme = forced;
+    public synchronized void setTheme(Theme theme) {
```

Comment:
This replaces `disableDarkTheme` and `forceDarkTheme`.  This allows the developer to present this option to the user and change it whenever they like.